soc/realtek/ec/rts5912/reg/reg_system.h

```diff
@@ -0,0 +1,334 @@
+/*
+ * Copyright (c) 2023 Realtek, SIBG-SD7
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef _REALTEK_RTS5912_REG_SYSTEM_H
```

Comment:
```suggestion
#ifndef ZEPHYR_SOC_REALTEK_RTS5912_REG_SYSTEM_H
```

Reply:
We changed the identifiers in `reg_system.h`, thanks.